profiling/src/profiling/mod.rs

```diff
@@ -650,14 +651,12 @@ impl Profiler {
                 let depth = frames.len();
                 let (wall_time, cpu_time) = CLOCKS.with(|cell| cell.borrow_mut().rotate_clocks());
 
-                let mut labels = Profiler::message_labels();
+                let labels = Profiler::message_labels();
+                let mut timestamp = 0;
```

Comment:
Is `0` a valid value for a timestamp?

Reply:
Yes and no ;-)
The `0` marks this as there is no timestamp. In https://github.com/DataDog/dd-trace-php/pull/2299/files#diff-3b610cb72989821b75d3c33815a7c7160a76e30418613cdeec640c855b9ca167R384 you can see that I just give this to `NonZeroI64::new()` which would return an Option (and in case it is `0`, that Option would be None). 
I could also have this an optional type (`Option<u64>`), but then I would have to make a check on `None` before calling `NonZeroI64::new()`, otherwise, that constructor would magically to the thing anyway.